### PR TITLE
feat[CX-2706]: add my collection artwork route

### PR DESCRIPTION
**Pull request by MrSltun**
The type of this PR is: **Feat**

This PR solves [CX-2706]

### Description
This PR adds a route for My Collection artwork details, plus a small sidequest to update a few dead links and content in docs :)

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-2706]: https://artsyproduct.atlassian.net/browse/CX-2706?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/MyCollectionArtwork/myCollectionArtworkRoutes.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    //       ...ArtworkApp_artwork
+    //     }
+    //     me {
+    //       ...ArtworkApp_me
+    //     }
+    //   }
+    // `,
+    // cacheConfig: {
+    //   force: true,
+    // },
```
Comment (MounirDhahri) [inline src/Apps/MyCollectionArtwork/myCollectionArtworkRoutes.tsx:40]: **issue(non-blocker):** I would suggest we remove this

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
cc @olerichter00

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 43912f7585a4650a0cea06af8e7702f049dad8f0
</p>

File: src/Apps/MyCollection/myCollectionRoutes.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const MyCollectionArtworkApp = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "myCollectionBundle" */ "./Routes/MyCollectionArtwork/MyCollectionArtworkApp"
+    ),
+  {
+    resolveComponent: component =>
+      component.MyCollectionArtworkAppFragmentContainer,
+  }
+)
```
Comment (olerichter00) [inline src/Apps/MyCollection/myCollectionRoutes.tsx:13]:
```suggestion
const MyCollectionArtwork = loadable(
  () =>
    import(
      /* webpackChunkName: "myCollectionBundle" */ "./Routes/MyCollectionArtwork/MyCollectionArtwork"
    ),
  {
    resolveComponent: component =>
      component.MyCollectionArtworkAppFragmentContainer,
  }
)
```

Comment (olerichter00) [reply src/Apps/MyCollection/myCollectionRoutes.tsx:13]: This is the pattern used in other apps. The routes inside don't get the `App` postfix.

**Review by olerichter00 — DISMISSED** [review]
Looks good! I just added a comment about the naming of the artwork route.

File: docs/best_practices.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ Within a sub-app, things are typically structured like so:
 
 If there's a component that might be shared across multiple sub-apps, it should go in the top-level [`/Components`](https://github.com/artsy/force/tree/1842553ad34475bc3b804f00c6410d7f23d64f65/src/Components) folder.
 
-Framework code is located in [`v2/Artsy`](https://github.com/artsy/force/tree/1842553ad34475bc3b804f00c6410d7f23d64f65/src/Artsy).
+Framework code is located in [`/System`](https://github.com/artsy/force/tree/main/src/System).
```
Comment (damassi) [inline docs/best_practices.md:101]: Ty @MrSltun for updating the docs, totally missed this 🙏

**Pull request merged**